contrib/snibbetracker/patches/8483d24a61703bcbab114673fae11ca4c5f0ef28.patch

```diff
@@ -25,7 +73,7 @@
 +COMPILER_FLAGS = -Wall -std=gnu99 -lasound -Wno-unused-function $(CFLAGS)
 +
 +# Linker flags
-+LINKER_FLAGS = -lSDL2 -lm $(LDFLAGS) $(CFLAGS)
++LINKER_FLAGS = -lSDL2 -lm $(LDFLAGS)
```

Comment:
this part changes things so cflags are not used during link sequence anymore, that's wrong